### PR TITLE
fix: fix port errors for k8s

### PR DESCRIPTION
**Pull request by Faraz32123**

### CODE REVIEW

File: tutorjupyterlite/patches/k8s-deployments [hunk 1]
```diff
@@ -23,6 +23,9 @@ spec:
           volumeMounts:
             - name: fss-pvc-content
               mountPath: /jupyterlite/{{ JUPYTERLITE_CONTENT_DIR }}
+          env:
```
Comment (Ali-Salman29) [inline tutorjupyterlite/patches/k8s-deployments:26]: Remove this now, as you already updated the Dockerfile.

File: tutorjupyterlite/templates/jupyterlite/build/jupyterlite/Dockerfile [hunk 2]
```diff
@@ -18,7 +18,7 @@ WORKDIR /jupyterlite
 RUN pip install -r requirements-full.txt
 {% endif %}
 
-ENV JUPYTERLITE_PORT 8000
+ENV JUPYTERLITE_PORT=8000
 ENV JUPYTERLITE_OUTPUT_DIR dist
```
Comment (Ali-Salman29) [inline tutorjupyterlite/templates/jupyterlite/build/jupyterlite/Dockerfile]: Also add `=` in this variable for the synchronization.

**Review by Ali-Salman29 — CHANGES_REQUESTED** [review]

**Review by ziafazal — APPROVED** [review]
👍

File: tutorjupyterlite/patches/k8s-deployments (hunk 1 above)
Comment (Faraz32123) [reply tutorjupyterlite/patches/k8s-deployments:26]: As discussed in meeting, we had to modify settings in both Dockerfile and K8s-deployments.
@wahajArbisoft @ziafazal

**Review by wahajArbisoft — APPROVED** [review]

**Review by Ali-Salman29 — APPROVED** [review]

**Pull request merged**